src/services/item/test/index.test.ts

```diff
@@ -556,10 +556,10 @@ describe('Item routes tests', () => {
         expect(response.statusCode).toBe(StatusCodes.OK);
 
         const data = response.json();
-        expect(data).toHaveLength(items.length);
+        expect(data.data).toHaveLength(items.length);
         items.forEach(({ id }) => {
           expectItem(
-            data.find(({ id: thisId }) => thisId === id),
+            data?.data?.find(({ id: thisId }) => thisId === id),
             items.find(({ id: thisId }) => thisId === id),
```

Comment:
It would be interesting to add a test to see if you indeed get the second page, and/or the limit works.